.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,33 @@
+name: Dockerize API
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
try to build it on all branches, don't limit it to master only

Reply:
ok